roles/rsyslog/README.md

```diff
@@ -338,7 +338,7 @@ WARNING: Pre-existing rsyslog.conf and configuration files in /etc/rsyslog.d are
 
 Describing how the configuration files are defined to be deployed using the viaq case.
 
-Viaq configuration files are defined in {{ rsyslog_viaq_rules }} in /roles/input_rols/viaq/defaults/main.yaml.  The set is made from the generic modules{rsyslog_conf_global_options, rsyslog_conf_local_modules, rsyslog_conf_network_modules, rsyslog_conf_common} and viaq specific configurations.
+Viaq configuration files are defined in {{ __rsyslog_viaq_rules }} in /roles/input_rols/viaq/defaults/main.yaml.  The set is made from the generic modules{rsyslog_conf_global_options, rsyslog_conf_local_modules, rsyslog_conf_network_modules, rsyslog_conf_common} and viaq specific configurations.
```

Comment:
If you are prefixing the variable name with underscores, it is internal and should it be documented then at all? (maybe it should, but in some developer/internal focused documentation, not in user documentation.) also, there is a typo in `input_rols`

Reply:
:p  thanks!

Reply:
The typo input_rols was fixed.
Documentation for the internal vars will be added in a separate pr.